graphql/server.go

```diff
@@ -86,6 +86,16 @@ type SafeError struct {
 	message string
 }
 
+type ClientError SafeError
+
+func (e ClientError) Error() string {
```

Comment:
do we need to redeclare Error() and SanitizedError()?

Reply:
strangely yes
```
graphql/server.go:100:20: cannot use ClientError literal (type ClientError) as type error in return argument:
	ClientError does not implement error (missing Error method)
```